pallets/gear/src/internal.rs

```diff
@@ -585,18 +617,50 @@ where
                 })
                 .unwrap_or_default();
 
+            let delay_gas_value = delay
```

Comment:
firstly convert into u64 to avoid overflows

Reply:
done